pkg/v1/tkg/client/common.go

```diff
@@ -144,6 +172,32 @@ func (c *TkgClient) validateDockerPrerequisites() error {
 	return nil
 }
 
+func (c *TkgClient) ValidateDockerResourcePrerequisites() error {
+	var dockerResourceCpus, dockerResourceTotalMemory int
+	var err error
+
+	// validate docker allocated CPU and memory against recommended minimums
+	if dockerResourceCpus, err = checkDockerResource("NCPU"); err != nil {
```

Comment:
nit: Can we initialize "NCPU" and "MemTotal" as string constants, and pass those constants?

Reply:
Ah yes, this is a question I forgot to ask when we synched over zoom. I just pushed a commit that initializes these as constants within the function. I meant to ask, is there a better place that we should initialize constants that would be consumed by this function...maybe somewhere we would import constants from. Or is it okay to just initialize within the function as I have in this commit.